### PR TITLE
merge master to release 2.2.0

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -97,7 +110,7 @@ set(GTEST_VERSION "1.11.0" CACHE STRING "googletest version in use" FORCE)
 
 set(OPAE_VERSION_LOCAL   "" CACHE STRING "OPAE local version")
 set(OPAE_VERSION_MAJOR    2 CACHE STRING "OPAE major version" FORCE)
-set(OPAE_VERSION_MINOR    2 CACHE STRING "OPAE minor version" FORCE)
+set(OPAE_VERSION_MINOR    3 CACHE STRING "OPAE minor version" FORCE)
```
Comment (tswhison) [inline CMakeLists.txt:113]: Be sure to reset the version to 2.2.0 after this goes in.  This change will set it to 2.3.0.

File: packaging/opae/version [hunk 2]
```diff
@@ -24,5 +24,5 @@
 # CONTRACT,  STRICT LIABILITY,  OR TORT  (INCLUDING NEGLIGENCE  OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-declare -r PROJECT_VERSION='2.2.0'
+declare -r PROJECT_VERSION='2.3.0'
```
Comment (tswhison) [inline packaging/opae/version:27]: Change back to 2.2.0 after this goes in..

File: packaging/opae.admin/version [hunk 3]
```diff
@@ -24,5 +24,5 @@
 # CONTRACT,  STRICT LIABILITY,  OR TORT  (INCLUDING NEGLIGENCE  OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-PROJECT_VERSION='2.2.0'
+PROJECT_VERSION='2.3.0'
```
Comment (tswhison) [inline packaging/opae.admin/version:27]: Change this back to 2.2.0 after this merge goes in.

**Review by tswhison — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (anandaravuri) [reply CMakeLists.txt]: fixed

File: packaging/opae/version (hunk 2 above)
Comment (anandaravuri) [reply packaging/opae/version]: fixed

File: packaging/opae.admin/version (hunk 3 above)
Comment (anandaravuri) [reply packaging/opae.admin/version]: fixed

**Review by tswhison — APPROVED** [review]

**Pull request merged**